### ISSUE TITLE
"eof-newline": false causes UnhandledPromiseRejectionWarning

### ISSUE THREAD

**Issue by userabuser** [csscomb/csscomb.js#575]
When `eof-newline` is set to `false` in config, I receive `UnhandledPromiseRejectionWarning`.

```
(node:7393) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): TypeError: ast.remove is not a function
(node:7393) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

When set to `true` in config, no exception.

When omitted from the config, no exception.

Config example:

```
{
    "remove-empty-rulesets": true,
    "always-semicolon": true,
    "color-case": "lower",
    "block-indent": "    ",
    "color-shorthand": false,
    "element-case": "lower",
    "eof-newline": false,
    "leading-zero": true,
    "quotes": "double",
    "sort-order-fallback": "abc",
    "space-after-colon": " ",
    "space-before-combinator": " ",
    "space-after-combinator": " ",
    "space-between-declarations": "\n",
    "space-before-opening-brace": " ",
    "space-after-opening-brace": "\n",
    "space-after-selector-delimiter": "\n",
    "space-before-selector-delimiter": "",
    "space-before-closing-brace": "\n",
    "strip-spaces": true,
    "tab-size": true,
    "unitless-zero": true,
    "vendor-prefix-align": true
}
```

I have tried various config arrangements, various target files with and without complex rule-sets and the problem persists.

Node: v7.10.1
OS: Windows 10
File: *.scss
Compile method: CLI

I have not tried this on OSX yet nor specifically dug through the source to determine the absolute problem.

Related:

#501 
#502

**Comment by ryanaltvater** [comment]
Any update(s)? I am also experiencing this only on Windows 10. I have not experienced this on macOS.

Node: 10.4.1
OS: Windows 10 (April Update)
File: *.html or *.scss
Compile method: Keyboard shortcut

**Comment by jirsbek** [comment]
I've got the same problem. The reason I've got `eof-newline` set to false is .editorconfig handles the newlines for me and also when cssbomb's newline is added caret is moved to the ending new line which is pretty annoying (using latest Atom a plugin csscomb-atom).

**Comment by jdalton** [comment]
Fixed by #552.
